app/boards/shields/sofle/sofle.dtsi

```diff
@@ -49,7 +49,7 @@ RC(3,0) RC(3,1) RC(3,2) RC(3,3) RC(3,4) RC(3,5) RC(4,5) RC(4,6) RC(3,6) RC(3,7)
         label = "LEFT_ENCODER";
         a-gpios = <&pro_micro 21 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
         b-gpios = <&pro_micro 20 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
-        resolution = <4>;
+        resolution = <2>;
```

Comment:
I believe this depends on the encoder hardware itself and not the shield. IMO it should be set the same for all shields (to the most common value if possible) then the user should override it in their config if their encoder is different.

Reply:
Ah, I didn't realize that. I just noticed my encoder was skipping steps.

No problem, I can remove that commit if that helps getting the changes merged.

Reply:
I have removed this now.